src/templates/storiesTemplate.ts

```diff
@@ -5,8 +5,7 @@ export function storiesTemplate(
   let text =
     `import React from 'react';\n` +
     `import {Meta, StoryFn} from '@storybook/react';\n\n` +
-    `import type {${componentName}Props} from './${componentName}';\n` +
-    `import {${componentName}} from './${componentName}';\n\n`;
+    `import {${componentName}, Props} from './${componentName}';\n\n`;
```

Comment:
We added the separate `type` import back in https://github.com/andrewmcgov/react-component-generator/pull/15. 

Was there a reason this was changed?